src/main/java/edu/ucsb/cs156/dining/entities/Review.java

```diff
@@ -0,0 +1,42 @@
+package edu.ucsb.cs156.dining.entities;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+/**
+ * This is a JPA entity that represents a restaurant.
+ */
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Entity(name = "reviews")
+public class Review {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+
+  private long reviewerId;
+  private String diningCommons;
+  private String meal;
+  private long itemId;
+  private String station;
+  private LocalDateTime dateServed;
+  private long stars;
+  private String reviewText;
+  private String status;
+  private long modId;
+  private String modComments;
+  private LocalDateTime createdDate;
+  private LocalDateTime lastEditedDate;
```

Comment:
This is storing a lot of redundant data.

The diningCommons, meal, and station do not need to be in this table.  There should just be an itemId that is the primary key for the DiningCommonsMenuItems table, which has

```
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private long id;
  private String diningCommons;
  private String meal;
  private String name;
  private String station;
```

It will still be possible for an api endpoint for reviews to return all of that information. But it's important, from a data integrity standpoint and an efficiency standpoint, not to store redundant information in relational database tables.  This is called "database normalization", and it's an important topic in CMPSC 174A.

See also: https://learn.microsoft.com/en-us/office/troubleshoot/access/database-normalization-description

Reply:
added change, I removed diningcommons, meal, and station